app/src/main/java/net/skyscanner/backpack/demo/compose/Text.kt

```diff
@@ -21,39 +21,118 @@ package net.skyscanner.backpack.demo.compose
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import net.skyscanner.backpack.compose.text.BpkText
 import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkDimension
+import net.skyscanner.backpack.demo.BackpackDemoTheme
 
+@Preview
 @Composable
-fun TypographyStylesStory() {
-  val styles = listOf(
-    BpkTheme.typography.hero1 to "Hero 1",
-    BpkTheme.typography.hero2 to "Hero 2",
-    BpkTheme.typography.hero3 to "Hero 3",
-    BpkTheme.typography.hero4 to "Hero 4",
-    BpkTheme.typography.hero5 to "Hero 5",
-    BpkTheme.typography.heading1 to "Heading 1",
-    BpkTheme.typography.heading2 to "Heading 2",
-    BpkTheme.typography.heading3 to "Heading 3",
-    BpkTheme.typography.heading4 to "Heading 4",
-    BpkTheme.typography.heading5 to "Heading 5",
-    BpkTheme.typography.subheading to "Subheading",
-    BpkTheme.typography.bodyLongform to "Body Longform",
-    BpkTheme.typography.bodyDefault to "Body Default",
-    BpkTheme.typography.footnote to "Footnote",
-    BpkTheme.typography.caption to "Caption",
-    BpkTheme.typography.label1 to "Label 1",
-    BpkTheme.typography.label2 to "Label 2",
-  )
-  LazyColumn(modifier = Modifier.padding(16.dp)) {
+fun HeroStyleStory() {
+  BackpackDemoTheme {
+    val styles = listOf(
+      BpkTheme.typography.hero1 to "Hero 1",
+      BpkTheme.typography.hero2 to "Hero 2",
+      BpkTheme.typography.hero3 to "Hero 3",
+      BpkTheme.typography.hero4 to "Hero 4",
+      BpkTheme.typography.hero5 to "Hero 5",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Preview
+@Composable
+fun HeadingStyleStory() {
+  BackpackDemoTheme {
+    val styles = listOf(
+      BpkTheme.typography.heading1 to "Heading 1",
+      BpkTheme.typography.heading2 to "Heading 2",
+      BpkTheme.typography.heading3 to "Heading 3",
+      BpkTheme.typography.heading4 to "Heading 4",
+      BpkTheme.typography.heading5 to "Heading 5",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Preview
+@Composable
+fun BodyStyleStory() {
+  BackpackDemoTheme {
+    val styles = listOf(
+      BpkTheme.typography.subheading to "Subheading",
+      BpkTheme.typography.bodyLongform to "Body Longform",
+      BpkTheme.typography.bodyDefault to "Body Default",
+      BpkTheme.typography.footnote to "Footnote",
+      BpkTheme.typography.caption to "Caption",
+      BpkTheme.typography.label1 to "Label 1",
+      BpkTheme.typography.label2 to "Label 2",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Composable
+private fun TypographyStylesStory(styles: List<Pair<TextStyle, String>>) {
+  LazyColumn(modifier = Modifier.padding(BpkDimension.Spacing.Base)) {
     items(styles) { style ->
-      Text(
+      BpkText(
         style = style.first,
         text = style.second
       )
     }
   }
 }
+
+@Preview
+@Composable
+fun DefaultText() {
+  BackpackDemoTheme {
```

Comment:
Can we add BackpackDemo theme to the base Story/Story composable?

Reply:
It is there - the theme is needed for the preview

Reply:
I'd remove the preview – we have an app for this :) In reality it's faster and easier to assemble the app then wait for preview :)

Reply:
I know they're working on making previews a lot faster - I find it quite helpful to see all different options in one place in the preview. The reason why it doesn't work in the preview is because we throw an error when somebody accesses typography/colour from the theme without something being wrapped in the `BpkTheme`